GUI/GUIPanel.cpp

```diff
@@ -318,6 +318,14 @@ void GUIPanel::OnMouseHover(int X, int Y, int Buttons, int Modifier)
 }
 
 
+/// <summary>
+/// Called when the mouse scroll wheel is moved.
+/// </summary>
+void GUIPanel::OnMouseWheelChange(int X, int Y, int Modifier, int MouseWheelChange)
```

Comment:
What's up with the empty method? Seems better to not implement it in GUIPanel at all so child classes have to, or if you want it here, why not define the implementation in the header?